main.js

```diff
@@ -1,9 +1,27 @@
 const buttons = document.querySelectorAll("button");
-const screen = document.getElementById("calculation_screen");
+let user_input = "";
+let calc_done = false;
 
 buttons.forEach(function (btn) {
   btn.addEventListener("click", (e) => {
     const button_clicked = e.currentTarget.textContent;
-    console.log(button_clicked);
+    if (calc_done && button_clicked !== "C") {
+      user_input = button_clicked;
+      document.querySelector("input").value = user_input;
+      calc_done = false;
+    } else if (button_clicked == "=") {
+      if (user_input !== "") {
+        let answer = eval(user_input);
```

Comment:
Of course using eval() here is going to be harmless but I just wanted you to look into the security concerns of using eval()  - this was actually one of the questions I got asked in an interview) :-)